packages/plus/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -2,54 +2,142 @@ import styled from '@emotion/styled'
 import { langs } from '@uiw/codemirror-extensions-langs'
 import { material } from '@uiw/codemirror-theme-material'
 import CodeMirror from '@uiw/react-codemirror'
-import { Text } from '@ultraviolet/ui'
-import type { ComponentProps } from 'react'
+import { consoleDarkTheme } from '@ultraviolet/themes'
+import { CopyButton, Label, Stack, Text } from '@ultraviolet/ui'
+import type { ComponentProps, ReactNode } from 'react'
 
-const StyledText = styled(Text)`
-  background-color: ${({ theme }) => theme.colors.neutral.backgroundStrong};
-  padding: ${({ theme }) => `${theme.space['1']} ${theme.space['2']}`};
-  border-radius: ${({ theme }) => `${theme.radii.default}`};
+const EditorContainer = styled.div`
+  position: relative;
   width: 100%;
 `
 
+const StyledCopyButton = styled(CopyButton)`
+  position: absolute;
+  top: ${({ theme }) => theme.space['1']};
+  right: ${({ theme }) => theme.space['1']};
+  z-index: 1;
```

Comment:
There is a risk while doing it that, if a modal appears on top of it the copy button will be above it. There should be a a better way to do it while not using z-index. For instance, if in the dom you added this element after the one you're trying to show it above, it should work